lessonplans/1_BayesianRefresher.md

```diff
@@ -0,0 +1,54 @@
+# Section 1: Bayesian refresher and Introduction to ArviZ
+
+* Walk students through a Bayesian Analysis and showcase all portions of the Bayesian workflow.  
+* Have students work together with instructor to gain confidence in seeing a full analysis.
+* Showcase mini tutorial on how to setup a bayesian model in PyMC3 and PyStan.
+* Explain how model fitting is easy with well written Probabilistic Programming
+* Languages, but just because the software fits a model, it doesn’t mean the model is a good one
+* Demonstrate how arrays of data are not well suited for communication or understanding
+
+## Activities
+### Instructor Do: Fit model and demonstrate ArviZ (15 minutes)
+* Briefly run through 8 schools model non-centered
+* Show how model fit may look good but there seem to be lots of divergences.
+* Then show centered parametrization and how the plots change.
+
+* Explain why eight schools is so important
+* Highlight link through 
+* Showcase link to Michael Betancourt's analysis, 8 schools explainer in ArviZ Docs,
+and [Everything I need to know about Bayesian Statistics](https://statmodeling.stat.columbia.edu/2014/01/21/everything-need-know-bayesian-statistics-learned-eight-schools/)
+
+
+### Student Do: Fit your own model (15 minutes)
```

Comment:
fit your own model in 15 minutes seems quite ambitious! it would perhaps be possible if we provide students with some datasets, and write down the models with statistical notation, or something like that? Choices of likelihoods and priors tend to be pattern matching for a long time (or always?)

Reply:
Ah yes, let's do this. What do you think of have students fit a very simple binomial model where we've specified the model in math notation? My motivation for this is partially about them understanding, and moreso about weeding out any tech issues in a simple debuggable activity very early on.